iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/ClearTextProtocolsCheck.java

```diff
@@ -47,9 +48,9 @@ private static Consumer<ContextualResource> propertyIsNotSetOrFalse(String prope
       .reportIf(isFalse(), GENERAL_ISSUE_MESSAGE);
   }
 
-  private static void checkFtpsState(ContextualResource resource) {
-    resource.property("ftpsState")
-      .reportIf(isEqual("AllAllowed"), GENERAL_ISSUE_MESSAGE);
+  private static Consumer<ContextualResource> propertyHasValue(String propertyName, String value) {
+    return resource -> resource.property(propertyName)
+      .reportIf(isEqual(value), GENERAL_ISSUE_MESSAGE);
```

Comment:
Maybe change the method name, the name make me think it's a Predicate method, while in fact it is reporting an issue.
Something like `reportIfPropertyHasValue` or `checkPropertyWithValue`?